### ISSUE TITLE
"extractTo" doesn't work

### ISSUE THREAD

**Issue by LiubavaL** [dmamontov/rararchiver#4]
Hi there!
I'm trying to extract rar-archive containing images, but extracted files seem to be damaged, their size are less then original.
My code:
`$rar = new RarArchiver($uploadedImages, RarArchiver::CREATE);`
`if ($rar->isRar()) {`
`             $rar->extractTo($path);`
`         }`
Could anybody help me? Thanks in advance!

**Comment by dmamontov** [comment]
Are you sure you have a rar?

**Comment by Underdog-01** [comment]
Compression works but not with add files.. I had to write my own sub-routine to copy files into a parent folder and then use buildFromDirectory to compress the entire folder which seems to work properly.
Also it puts the files in a rar archive but there is little or no compression.

LiubavaL is correct.
All decompressed rar files are corrupted.
It looks like it prematurely cuts the file prior to gathering all of its data.
I suspect there is an issue in the extractTo function.

Unfortunately I had to use pecl-rar to decompress files because I can not get the decompression working.
This is a pity because I wanted to provide rar decompression support without that dependency.

@dmamontov:
I suggest you test decompression of archives created with Winrar and not ones created with this compression class. Perhaps doing that will help you reproduce the issue.
